examples/streaming/sol/src/test/scala/org/apache/gearpump/streaming/examples/sol/SOLSpec.scala

```diff
@@ -46,7 +44,7 @@ class SOLSpec extends PropSpec with PropertyChecks with Matchers with BeforeAndA
       "-streamProducer", "1",
       "-streamProcessor", "1",
       "-bytesPerMessage", "100",
-      "-stages", "2")
+      "-stages", "10")
```

Comment:
to cover this line

```scala
    computation = 0.until(stages - 2).foldLeft(computation) { (c, id) =>
      c ~ partitioner ~> streamProcessor.copy()
    }
```
in case future changes may cause similar issues